data/countries/de/ni/aurich-norden/data.yaml

```
@@ -16,17 +18,21 @@ district: Aurich-Norden
 city: Aurich
 urls:
   - type: WEBSITE
-    url: http://www.gruene-ov-aurich.de/
+    url: https://www.gruene-aurich.org/
```

```suggestion
    url: https://gruene-aurich.org/
```

I land on `https://gruene-aurich.org/`